src/models/weatherData/pressure.js

```diff
@@ -1,7 +1,13 @@
 export default class Pressure {
   constructor(inSightPressureData) {
-    this.max = inSightPressureData.mx;
-    this.min = inSightPressureData.mn;
-    this.avg = inSightPressureData.av;
+    try {
+      this.max = inSightPressureData.mx;
+      this.min = inSightPressureData.mn;
+      this.avg = inSightPressureData.av;
+    } catch (Exception) {
```

Comment:
This is a great idea! We'd generally just use an `e` in our try-catch blocks for simple cases like this where we aren't doing anything with the exception:
```
try {
    ...
} catch (e) {
    ...
}
```